rss_test/rss_list/management/commands/loader.py

```diff
@@ -24,37 +29,32 @@ def to_date(value):
 
 
 def store_single_feed(item):
-    try:
-        f = Feed.objects.get(link=item['link'])
-    except ObjectDoesNotExist:
-        f = Feed(title=item['title'], link=item['link'], pub_date=to_date(item['published']))
-        f.save()
-        return 1 # 'success'
-    return 0     # 'feed already in database'
+    _, created = Feed.objects.get_or_create(
+        link=item['link'],
+        defaults={'title': item['title'], 'pub_date': to_date(item['published'])},
+    )
+    return created
```

Comment:
Częsty przypadek więc django ma na to swoją funkcjię. Created jest Boolem, ale True i False to dalej liczby 1 i 0.